tool/grind.dart

```diff
@@ -61,7 +62,7 @@ void issues() {
       } on TypeError catch (_) {
         message = issueType.message(argsWithArray);
       }
-      sb.writeln('|${issueType.code}|$message|'
+      sb.writeln('|${issueType.code}|${message.replaceAll('\n', ' ')}|'
```

Comment:
I wondered about the options of 

- removing the `\n` here
- omitting it at https://github.com/KhronosGroup/glTF-Validator/blob/9ecda14d6d2899d3680182ca93c6aea95a1662c3/lib/src/errors.dart#L248 in the first place

(The "Parser output" info from the `invalidJson` does not have the `\n`...). 
But don't have any basis to say what would be "better". I could imagine that, depending on the output, the `\n` adds clarity or readability.

Reply:
The invalid URI error message consists of two lines, so prepending `\n` makes sense there although the layout of that output format makes sense only with monospace fonts.

In any case, ensuring that the table does not contain extra line breaks (even if there aren't any) is an improvement.